docs/components/modeler/bpmn/terminate-events/terminate-events.md

```diff
@@ -0,0 +1,48 @@
+---
+id: terminate-events
+title: "Terminate events"
```

Comment:
🔧  In the BPMN coverage table we refer to them as Termination events. Here we call it Terminate events. Let's align it and rename this page to Termination events, or rename it in the table to Terminate events.

Reply:
This is inconsistent across our web presence, which I'll have @christinaausley coordinate and chase down when she's back. Thanks for spotting this!

**Terminate event**(s) or **terminate** instead of termination.

As seen in the [spec](https://www.omg.org/spec/BPMN/2.0.2/PDF).

Reply:
I updated this in the latest commit, but for anything else in our docs, I'll create an issue for @christinaausley to investigate. Thanks!